src/app/containers/Toggle/README.md

```diff
@@ -0,0 +1,48 @@
+# Toggle Container
+
+## Overview
+
+The toggle container is a component that is designed to wrap other components and render the child component based on a feature toggle.
+
+## Feature Toggles
+
+At the time of writing feature toggles are designed to be global (i.e not service specific) and are dependant on the application environment.
+feature toggles can be found in `src/app/lib/config/toggles`
+
+## Useage
+
+To conditionally render a component based on the value of a toggle you can either wrap the child component or use the custom `useToggle` hook.
+
+### Wrapping a child component
+
+```
+<Toggle toggleName="foo">
+    <ChildComponent>
+        <p>I will be rendered if the enabled value of toggle: foo is set to true</p>
+    </ChildComponent>
+</Toggle>
+```
+
+### Rendering a fallback UI component if the toggle value is set to false
+
+```
+<Toggle toggleName="foo" fallbackComponent={<h1>I am the fallback UI</h1>}>
```

Comment:
Just wondering if a render prop component is maybe a more appropriate API here?

Also, why provide 2 methods of using toggles? When would we use one over the other? Could we get away with using only the `useToggle` hook?

Reply:
The reason for their being two approaches is that in some scenarios I can see time where we might not want to wrap a component to out it behind a feature toggle. But rather just grab the boolean value of the toggle and let the component handle the conditional rendering.

Where as sometimes the wrapping component would prove a more simple/readable method of using a feature toggle and in my eyes a better way to pass in a fallback UI component.

Either way the wrapping container uses the useToggle hook.